src/CoreFoundation/Dispatch.cs

```diff
@@ -556,11 +556,16 @@ public DispatchQualityOfService GetQualityOfService (out int relative_priority)
 			}
 		}
 
+		static IntPtr dispatch_queue_create_wrapper (string label, IntPtr attr)
```

Comment:
NBD, but I think this should work due to method overloading:

```suggestion
		static IntPtr dispatch_queue_create (string label, IntPtr attr)
```